manifest.json

```diff
@@ -6,5 +6,10 @@
   "categories": ["Advertising"],
   "provides": ["events"],
   "allowCustomFields": true,
-  "permissions": {}
+  "permissions": {
+    "access_client_kv": {
+      "description": "",
```

Comment:
Let's add a description that explains the need for that permission

Reply:
Updated :)